pkg/apis/eventing/v1beta2/eventtype_defaults.go

```diff
@@ -23,7 +23,5 @@ func (et *EventType) SetDefaults(ctx context.Context) {
 }
 
 func (ets *EventTypeSpec) SetDefaults(ctx context.Context) {
-	if ets.Broker == "" {
-		ets.Broker = "default"
-	}
+	// noop for now
```

Comment:
```suggestion
	if ets. Reference == nil {
		ets.Reference = &duckv1.KReference{
					APIVersion: "eventing.knative.dev/v1",
					Kind:       "Broker",
					Name:       "default",
				},
	}
```

Since we are setting broker to `default` when it is not set. Should we do this now instead?